docs/Capabilities/Integrations/Event-Subscription.md

```diff
@@ -1,10 +1,8 @@
 [Home](index) > [Capabilities](Capabilities) > [Integrations](Integrations) > **Event Subscription** 
 ***  
 
-![img](./images/coming-soon.png)
-
 > This page describe how to trigger a call to a webhook when a new submission event occurs for your form  
 
 ![event-subscription](images/event-sub.png)
 ***
-[Terms of Use](Terms-of-Use) | [Privacy](Privacy) | [Security](Security) | [Service Agreement](Service-Agreement) | [Accessibility](Accessibility)
\ No newline at end of file
+[Terms of Use](Terms-of-Use) | [Privacy](Privacy) | [Security](Security) | [Service Agreement](Service-Agreement) | [Accessibility](Accessibility)
```

Comment:
If the original file was formatter with Prettier, it looks like the GitHub editor wants it a different way. Maybe we shouldn't use the GitHub UI to make changes? Thoughts?

Reply:
Unrelated, but why do we have a footer on every page? Is it needed? Is there a less repetitive way of doing it?

Reply:
> Unrelated, but why do we have a footer on every page? Is it needed? Is there a less repetitive way of doing it?

Looks like not out of the box in markdown, unless use of some external script. Not sure if backstage has something for it.